widgets/deploymentsView/src/columns.tsx

```diff
@@ -37,24 +72,17 @@ const partialDeploymentsViewColumnDefinitions: Record<
     status: {
         width: '20px',
         render(deployment) {
-            const { Icon, Popup } = Stage.Basic;
-            const deploymentStatusIconProps: Record<DeploymentStatus, Pick<IconProps, 'name' | 'color'> | undefined> = {
+            const deploymentStatusIconProps: Record<DeploymentStatus, StatusIconName | undefined> = {
                 [DeploymentStatus.Good]: undefined,
-                [DeploymentStatus.InProgress]: { name: 'spinner', color: 'orange' },
-                [DeploymentStatus.RequiresAttention]: { name: 'exclamation', color: 'red' }
+                [DeploymentStatus.InProgress]: 'inProgress',
+                [DeploymentStatus.RequiresAttention]: 'requiresAttention'
```

Comment:
If you simplify the code (see my first comment), you should be able to avoid using strings and have something like:
```suggestion
                [DeploymentStatus.InProgress]: statusIcon.inProgress,
                [DeploymentStatus.RequiresAttention]: statusIcon.requiresAttention
```

Reply:
I'm not addressing this one and the one below yet, to see your reply to my arguments above :slightly_smiling_face:

Reply:
Not relevant anymore. Resolving.